src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.ts

```diff
@@ -61,14 +63,19 @@ export class CephfsSubvolumeListComponent extends CdForm implements OnInit, OnCh
   selectedName: string = '';
 
   subVolumes$: Observable<CephfsSubvolume[]>;
+  subVolumeGroups$: Observable<CephfsSubvolumeGroup[]>;
   subject = new ReplaySubject<CephfsSubvolume[]>();
 
+  subVolumeGroupName: string = '';
+  activateButtonIndex: number = 0;
```

Comment:
We can remove storing the index and use the name for comparing it.
```suggestion
  activeSubvolumeGroupName: string = '';
```